*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:42439 (IPv4)
Extension host agent listening on 42439

[20:12:39] 




[20:12:39] Extension host agent started.
[20:12:39] Started initializing default profile extensions in extensions installation folder. file:///home/ubuntu/.vscode-server/extensions
[20:12:39] Completed initializing default profile extensions in extensions installation folder. file:///home/ubuntu/.vscode-server/extensions
[20:12:39] [127.0.0.1][efd030af][ManagementConnection] New connection established.
[20:12:39] [127.0.0.1][feadb188][ExtensionHostConnection] New connection established.
[20:12:40] [127.0.0.1][feadb188][ExtensionHostConnection] <1415> Launched Extension Host Process.
[20:16:47] [127.0.0.1][efd030af][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[20:16:47] [127.0.0.1][feadb188][ExtensionHostConnection] <1415> Extension Host Process exited with code: 0, signal: null.
Cancelling previous shutdown timeout
[20:16:47] Cancelling previous shutdown timeout
Last EH closed, waiting before shutting down
[20:16:47] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[20:16:50] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[20:16:50] Cancelling previous shutdown timeout
[20:16:50] [127.0.0.1][5e992966][ManagementConnection] New connection established.
[20:16:50] [127.0.0.1][1b782d7c][ExtensionHostConnection] New connection established.
[20:16:50] [127.0.0.1][1b782d7c][ExtensionHostConnection] <1583> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at A.U (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:43:32407)
    at A.updateTitle (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:43:27539)
    at U.<computed> (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:43:22765)
    at Object.call (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at $.s (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at $.q (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at d.y (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at d.fire (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ne (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:25:10072)
    at process.emit (node:events:514:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[20:16:52] Error [CodeExpectedError]: Could not find pty on pty host
    at A.U (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:43:32407)
    at A.updateTitle (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:43:27539)
    at U.<computed> (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:43:22765)
    at Object.call (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:34:5041)
    at $.s (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:32:5153)
    at $.q (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:32:4662)
    at a.value (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:32:4064)
    at d.y (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at d.fire (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process.ne (/home/ubuntu/.vscode-server/bin/05047486b6df5eb8d44b2ecd70ea3bdf775fd937/out/vs/platform/terminal/node/ptyHostMain.js:25:10072)
    at process.emit (node:events:514:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
New EH opened, aborting shutdown
[20:21:50] New EH opened, aborting shutdown
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5)
[22:16:04] AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:5000
        at afterConnectMultiple (node:net:1520:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 5000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5)
[22:16:04] AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:5000
        at afterConnectMultiple (node:net:1520:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 5000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5)
[23:45:21] AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:5000
        at afterConnectMultiple (node:net:1520:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 5000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5)
[23:45:21] AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:5000
        at afterConnectMultiple (node:net:1520:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 5000
    }
  ]
}
rejected promise not handled within 1 second: AggregateError
stack trace: AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5)
[23:45:30] AggregateError
    at internalConnectMultiple (node:net:1082:18)
    at afterConnectMultiple (node:net:1533:5) {
  code: 'ECONNREFUSED',
  [errors]: [
    Error: connect ECONNREFUSED 127.0.0.1:5000
        at afterConnectMultiple (node:net:1520:16) {
      errno: -111,
      code: 'ECONNREFUSED',
      syscall: 'connect',
      address: '127.0.0.1',
      port: 5000
    }
  ]
}
